status_handler.go

```diff
@@ -40,14 +41,52 @@ type ErrorStatusHandler struct {
 	Component
 }
 
-// Handle generic error reponses.
+// Handle generic error responses.
 func (*ErrorStatusHandler) Handle(response *Response, _ *Request) {
 	message := map[string]string{
 		"error": http.StatusText(response.GetStatus()),
 	}
 	response.JSON(response.GetStatus(), message)
 }
 
+// RequestErrorStatusHandler a generic (error) status handler for requests.
+type RequestErrorStatusHandler struct {
```

Comment:
Renaming it to `BadRequest` specifically will constrain the messages we can put in there. What do you think of a more generic `RequestStatusHandler`?

Reply:
What other kind of errors are you thinking about for this status handler? Do you think it could be used for other status codes than 400?

Reply:
Yes several. I would like to believe that users of Goyave can also use this for their own handling. Not only 400, but 406, 412, 417, 422 etc. Maybe we don't implement it, but others may. Then this can be used without them having to built their own struct/handler.

I'm fine with `RequestStatusHandler`?

Reply:
We only check for `ExtraParseError` in this handler so maybe we could name it `ParseErrorStatusHandler`?